walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -31,44 +32,53 @@
   private List<List<String>> optionalAttendeesCombinations;
 
   /** 
-   * Finds a list of times when the requested event can happen, given the list of current events and the request information.
-   * An event includes a list of attendees, a title, and a time range of the event.
-   * The request includes lists of mandatory and optional attendees (either list can be empty) and a duration of the meeting in minutes.
+   * Finds a list of times when the requested event can happen, given the list
+   * of current events and the request information. An event includes a list of
+   * attendees, a title, and a time range of the event. The request includes
+   * lists of mandatory and optional attendees (either list can be empty) and a
+   * duration of the meeting in minutes.
    */
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
```

Comment:
So sorry to keep doing this. We use 100 characters a line at Google. I just double checked.

Reply:
lmao that's ok. I'll just change the tool's setting